src/types/dict.rs

```diff
@@ -430,7 +438,9 @@ enum DictIterImpl {
 
 impl DictIterImpl {
     #[inline]
```

Comment:
It's more typical to have the doc comments before the attribute; I'll move the attribute down.

```suggestion
```